docs/docs/20-usage/21-multi-pipeline.md

```diff
@@ -1,11 +1,10 @@
+# Multi pipelines
 
-# Multi-pipeline builds
+> NOTE: This Feature is only available for Github & Gitea repositories. Follow [this](https://github.com/woodpecker-ci/woodpecker/issues/131) issue to support further development.
```

Comment:
```suggestion
> NOTE: This Feature is only available for GitHub & Gitea repositories. Follow [this](https://github.com/woodpecker-ci/woodpecker/issues/131) issue to support further development.
```

Might want to find and replace as there are lots of cases of this in the existing documentation too.

Reply:
I changed it for the complete documentation.